net/udpbroadcastrelay/src/opnsense/service/conf/actions.d/actions_udpbroadcastrelay.conf

```diff
@@ -0,0 +1,30 @@
+[start]
+command:
+    /usr/local/opnsense/scripts/OPNsense/UDPBroadcastRelay/setup.sh;
+    /usr/local/etc/rc.d/os-udpbroadcastrelay start
+parameters:%s
```

Comment:
Does start/stop/etc really expect a parameter?

Reply:
Erm.. I thought it did, apparently it doesn't - removed.

Reply:
Yes it does for services. Along with another fix in inc/plugins.inc.d the items in the services widget now start and stop independently.